src/Microsoft.Health.SqlServer.Api.UnitTests/Features/CompatibilityVersionHandlerTests.cs

```diff
@@ -29,7 +28,7 @@ public CompatibilityVersionHandlerTests()
     {
         _schemaMigrationDataStore = Substitute.For<ISchemaDataStore>();
         var collection = new ServiceCollection();
-        collection.Add(sp => new CompatibilityVersionHandler(_schemaMigrationDataStore)).Singleton().AsSelf().AsImplementedInterfaces();
+        collection.Add(_ => new CompatibilityVersionHandler(_schemaMigrationDataStore)).Singleton().AsSelf().AsImplementedInterfaces();
```

Comment:
I don't know if this does anything lol. Maybe it's changed, but this now might be a variable called `_`. Although, this totally may be a discard now in recent C# versions

Reply:
Yea... just found this documentation "For backwards compatibility, if only a single input parameter is named _, then, within a lambda expression, _ is treated as the name of that parameter.". I still think that it is slightly more readable in that we don't expect to use this passed in variable.